app/views/static/_deferred_posts.html.erb

```diff
@@ -0,0 +1,3 @@
+<script>
+  var ___deferred_posts = <%= raw DeferredPosts.dump.to_json %>;

```

Comment:
Though this is the default behavior for global scripts, I would suggest you explicitly assign this to `window` (`window.__deferredPosts = ...`)

Reply:
Agreed, since it resolves the ambiguity.